analyzers/src/SonarAnalyzer.Common/Rules/BypassingAccessibilityBase.cs

```diff
@@ -26,16 +28,22 @@ public abstract class BypassingAccessibilityBase<TSyntaxKind> : SonarDiagnosticA
         where TSyntaxKind : struct
```

Comment:
I think the code smell raised here makes sense and we should make this a concrete class with a protected constructor.

Reply:
It dissapeared with Pavels tracker refactoring.